### ISSUE TITLE
Plugin fails to load variables in IntelliJ

### ISSUE THREAD

**Issue by cjdudleybw** [Ashald/EnvFile#125]
I've run into a bug using EnvFile in IntelliJ Ultimate Edition (full version/specs below). The file definitely exists on my system, and I've checked with colleagues that the rest of the config is correct, however the plugin fails to load in the Environment Variables stored in my env file. 
Bulk pasting the contents of the file manually works fine, which is why I suspect a bug in the plugin. Is this a known issue? 

IntelliJ IDEA 2020.2.3 (Ultimate Edition)
Build #IU-202.7660.26, built on October 6, 2020
Runtime version: 11.0.8+10-b944.34 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.4.0-53-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 2988M
Cores: 12
Non-Bundled Plugins: PlantUML integration, claims.bold.intellij.avro, com.andrey4623.rainbowcsv, com.markskelton.one-dark-theme, com.potterhsu.jsonviewer, com.thvardhan.gradianto, com.vincentp.gruvbox-theme, de.netnexus.camelcaseplugin, net.ashald.envfile, net.seesharpsoft.intellij.plugins.csv, com.viartemev.requestmapper, org.intellij.scala
Current Desktop: X-Cinnamon

**Comment by goosmurf** [comment]
@cjdudleybw not sure if I ran into the same issue but for reasons still unknown, a run configuration that I had been using for years also just started failing with the same symptoms. Config looks correct but EnvFile was not loading the variables stored in my env file.

In my case I was able to workaround by creating a new run config with the same settings. Would be interested to know if that works for you?

**Comment by mfaizanse** [comment]
I am facing the same issue. The plugin is not loading the Environment Variables defined in my env file.

```
IntelliJ IDEA 2020.2.3 (Ultimate Edition)
Build #IU-202.7660.26, built on October 6, 2020
Licensed to IntelliJ IDEA Evaluator
Expiration date: December 23, 2020
Runtime version: 11.0.8+10-b944.34 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.4.0-53-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 949M
Cores: 4
Non-Bundled Plugins: net.ashald.envfile, com.intellij.kubernetes, com.intellij.bigdatatools, training
Current Desktop: ubuntu:GNOME
```

**Comment by virtuoushub** [comment]
@goosmurf I also ran into the same issue w/ a run config that had been working for months.

I tried recreating the run config, but that did not work.

```
IntelliJ IDEA 2020.2.4 (Community Edition)
Build #IC-202.8194.7, built on November 24, 2020
Runtime version: 11.0.9+11-b944.49 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 10.16
GC: ParNew, ConcurrentMarkSweep
...
Non-Bundled Plugins: net.ashald.envfile, org.jetbrains.kotlin
```

**Comment by ashald** [comment]
Hi there, plugin maintainer here. As you can tell, I have very limited time for supporting the plugin right now.
Funny enough, I use plugin in IDEA for JUnit run configurations on daily basis on macOS Catalina and that have been working for me flawlessly.

I haven't tried upgrading to Big Sure yet, nor I have Windows or Linux machines handy to try it there.
I just downloaded the latest version of IDEA, created a new project and confirmed plugin working.

Given the above, I'd assume that very likely is that plugin on its own works fine, but either project configuration got screwed up (EnvFile plugin on its own, IDEA version upgrade, a 3rd party plugin etc), or something else happened. Would be curious if @goosmurf 's workaround would work for anyone else. And if no... you know, we're all developers, right? The best way to get this fixed, is to debug the plugin when the issue is reproduced. The [README](https://github.com/ashald/EnvFile#building) has instructions on building the plugin, so you should be able to get run the plugin in debug mode with another instance of IDEA and catch the root cause of the issue. This way you'd bea bel to help yourself and others! 😊

**Comment by vojkny** [comment]
Not working on IDEA 2020.2.3.

**Comment by justin-dd** [comment]
Not that it's helping, but I updated to IDEA 2020.2.3 and it's also now not working for me - tried deleting the `.idea` directory, but that didn't seem to help.

**Comment by FedericoOldrini** [comment]
It seems to be something relative also to kotlin plugin 1.4.x. Rolling back to 1.3.latest the env plugin start working again.

same behaviour on 2020.2.3 and 2020.3

**Comment by Thource** [comment]
I had this same issue, I've upgraded Kotlin plugin to 1.4.21 (203-1.4.21-release-IJ6682.9) and EnvFile is now working as intended.

**Comment by ashald** [comment]
So I guess it's working now?

**Comment by mdassow-varde** [comment]
I'm still getting this issue in Pycharm with no Kotlin plugin installed

**Comment by sasha-wing** [comment]
Happens to me as well with PyCharm 2022.2.2 with no Kitlin plugin installed

**Comment by ashald** [comment]
This was supposed to be fixed with `3.4.1`. If you still face issues, open a new issue and note down your IDE version, your plugin version and run configuration in question.

**Comment by HWiese1980** [comment]
Regression? I have this issue with release _2022.3_ in IntelliJ IDEA Build #IU-223.8617.56, built on January 26, 2023. 

I'll open another issue
